hazelcast-client-legacy/src/main/java/com/hazelcast/client/impl/ClusterAuthenticator.java

```diff
@@ -60,7 +60,7 @@ public void authenticate(ClientConnection connection) throws AuthenticationExcep
         //contains remoteAddress and principal
         SerializableList collectionWrapper;
         final ClientInvocation clientInvocation = new ClientInvocation(client, auth, connection);
-        final Future<SerializableList> future = clientInvocation.invoke();
+        final Future<SerializableList> future = clientInvocation.invokeUrgent();
```

Comment:
Ok than. I don't want to spend time on different solution on new and old client. Old client will be deleted soon anyway.